google-cloud-recaptcha_enterprise/samples/README.md

```diff
@@ -0,0 +1,61 @@
+<img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
+
+# Google Cloud Recaptcha Enterprise Ruby Samples
+
+This directory contains samples for google-cloud-recaptcha_enterprise. [Google Cloud Recaptcha Enterprise][recaptcha_docs]
+
+[recaptcha_docs]: https://cloud.google.com/recaptcha-enterprise
```

Comment:
If you want to use link references, that's fine but please be consistent in this file. Currently it looks like the rest of the links below are inline. Either make them all inline links or all references. (And if you want to use references, probably best to group all references at the very bottom of the file so they're easy to find.)

Reply:
Din't particularly wanted to use references added it that way as we have like that in other readme, updated it to inline reference.